Crawler/PartOfSpeechTagger/PosTagger.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Crawler.ExtensionMethods;
+using Crawler.LexicalAnalyzer;
+using Newtonsoft.Json;
```

Comment:
There is builtin json deserializer in .net core, System.Text.Json

Reply:
It's something to consider, but first I need to ensure it has the same functionality that Newtonsoft.Json have.